README.rst

```diff
@@ -34,9 +34,15 @@ are omitted.
     permissions, with a command such as ``gpasswd --add $(id -u) docker &&
     newgrp``.
 
+``fixtures/drpm-unsigned``
+    The ``createrepo`` and ``makedeltarpm`` utilities must be available.
+
 ``fixtures/rpm``
     The ``createrepo`` and ``modifyrepo`` utilities must be available.
 
+``fixtures/rpm-unsigned``
+    The ``createrepo``, ``modifyrepo`` and ``rpm-sign`` utilities must be available.
```

Comment:
There is no executable named `rpm-sign`.  However, there is an executable named `rpmsign`. Do you mean that? (Confusingly, the `rpmsign` utility is provided by the `rpm-sign` package.)